modules/oadp-deleting-backups-using-oc.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-deleting-backups-doc.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-deleting-backups-using-oc_{context}"]
+= Deleting a backup by creating a `DeleteBackupRequest` CR
+
+.Prerequisites
+
+* You have run a backup of your application.
+
+.Procedure
+
+. Create a `DeleteBackupRequest` CR manifest file.
++
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: DeleteBackupRequest
+metadata:
+  name: deletebackuprequest
+  namespace: openshift-adp
+spec:
+  backupName: <backup_name> # <1>
+----
+<1> Specify the name of the backup.
+
+. Apply the `DeleteBackupRequest` CR to delete the backup. 
++
+[source,terminal]
+----
+$ oc apply -f <deletebackuprequest_cr_filename> -n openshift-adp
```

Comment:
I think we can remove the namespace from apply command as we are already specified it in CR.
```suggestion
$ oc apply -f <deletebackuprequest_cr_filename> 
```